CrowdStrike/crowdstrike_falcon/ingest/parser.yml

```
@@ -82,6 +83,10 @@ stages:
           process.start: "{{parsed_event.message.event.ProcessStartTime | string | to_rfc3339}}"
         filter: "{{parsed_event.message.event.ProcessStartTime > 0}}"
 
+      - set:
+          process.hash.sha256: "{{parsed_event.message.event.IOCValue}}"
```

I wonder if `threat.indicator.file.hash.sha256` is not more suitable, with [`threat.indicator.type`](https://www.elastic.co/guide/en/ecs/8.6/ecs-threat.html#field-threat-indicator-type) defined to `file`:
```suggestion
          threat.indicator.file.hash.sha256: "{{parsed_event.message.event.IOCValue}}"
          threat.indicator.type: "file"
```